documentation/general/dotnet-run-file.md

```diff
@@ -47,6 +47,10 @@ Additionally, the implicit project file has the following customizations:
 
   - `FileBasedProgram` property is set to `true` and can be used by SDK targets to detect file-based apps.
 
+  - `DisableDefaultItemsInProjectFolder` property is set to `true` which results in `EnableDefaultItems=false` by default
```

Comment:
I don't think I'm understanding this documentation. I expected us to have a default behavior of deciding per-folder, whether to gather default items from it, based on whether it includes a project file. Is that what this is saying?

Reply:
The next line should be saying that:
> in case there is a project or solution in the same directory as the file-based app.

Reply:
I think I may have just misunderstood at a previous stage what solution we were actually proposing here. If we are treating this as essentially a best effort to avoid globbing up the world, I am ok with the current approach in this PR. With the understanding that it won't work for every possible repo configuration out there.

Reply:
> If we are treating this as essentially a best effort to avoid globbing up the world

Yes, that's what the current PR is about. We can improve in the future, but we would need to do some more complicated (and perf intensive) globbing to find the folders to exclude.